### ISSUE TITLE
qlippernetwork.cpp: no matching function for call to QlipperItem::toClipboard()

### ISSUE THREAD

**Issue by micrococo** [pvanek/qlipper#89]
When ENABLE_NETWORK_CLIPBOARD_SHARING is ON, build fails:
```
/home/tux/Descargas/qlipper/src/qlippernetwork.cpp: In member function 'void QlipperNetwork::readData()':
/home/tux/Descargas/qlipper/src/qlippernetwork.cpp:104:26: error: no matching function for call to 'QlipperItem::toClipboard()'
         item.toClipboard();
                          ^
In file included from /home/tux/Descargas/qlipper/src/qlippernetwork.cpp:21:0:
/home/tux/Descargas/qlipper/src/qlipperitem.h:62:10: note: candidate: void QlipperItem::toClipboard(const Actions&) const
     void toClipboard(const Actions & actions) const;
          ^~~~~~~~~~~
/home/tux/Descargas/qlipper/src/qlipperitem.h:62:10: note:   candidate expects 1 argument, 0 provided
make[2]: *** [CMakeFiles/qlipper.dir/build.make:266: CMakeFiles/qlipper.dir/src/qlippernetwork.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/qlipper.dir/all] Error 2
make: *** [Makefile:152: all] Error 2

```

**Comment by softwaresale** [comment]
I believe that I fixed this. I am submitting a pull request now.

**Comment by palinek** [comment]
fixed in #94
